packages/grafana-data/src/types/fieldOverrides.ts

```diff
@@ -15,6 +15,26 @@ export interface ConfigOverrideRule {
   properties: DynamicConfigValue[];
 }
 
+/**
+ * Describes config override rules created when interacting with Grafana.
+ *
+ * @public
```

Comment:
Maybe we should consider this an internal thingy?

Reply:
Ok ok, I will add that.